src/import-export/azext_import_export/generated/_params.py

```diff
@@ -0,0 +1,74 @@
+        c.argument('client_tenant_id', help='The tenant ID of the client making the request.')
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
+        c.argument('tags', tags_type)
+        c.argument('storage_account_id', help='The resource identifier of the storage account where data will be imported to or exported from.')
+        c.argument('type', help='The type of job')
+        c.argument('return_address', action=AddReturnAddress, nargs='+', help='Specifies the return address information for the job.')
+        c.argument('return_shipping', action=AddReturnShipping, nargs='+', help='Specifies the return carrier and customer\'s account with the carrier.')
+        c.argument('shipping_information', action=AddShippingInformation, nargs='+', help='Contains information about the Microsoft datacenter to which the drives should be shipped.')
+        c.argument('delivery_package', action=AddDeliveryPackage, nargs='+', help='Contains information about the package being shipped by the customer to the Microsoft data center.')
+        c.argument('return_package', action=AddReturnPackage, nargs='+', help='Contains information about the package being shipped by the customer to the Microsoft data center.')
+        c.argument('diagnostics_path', help='The virtual blob directory to which the copy logs and backups of drive manifest files (if enabled) will be stored.')
+        c.argument('log_level', help='Default value is Error. Indicates whether error logging or verbose logging will be enabled.')
+        c.argument('backup_drive_manifest', arg_type=get_three_state_flag(), help='Default value is false. Indicates whether the manifest files on the drives should be copied to block blobs.')
+        c.argument('state', help='Current state of the job.')
+        c.argument('cancel_requested', arg_type=get_three_state_flag(), help='Indicates whether a request has been submitted to cancel the job.')
+        c.argument('percent_complete', help='Overall percentage completed for the job.')
+        c.argument('incomplete_blob_list_uri', help='A blob path that points to a block blob containing a list of blob names that were not exported due to insufficient drive space. If all blobs were exported successfully, then this element is not included in the response.')
+        c.argument('drive_list', action=AddDriveList, nargs='+', help='List of up to ten drives that comprise the job. The drive list is a required element for an import job; it is not specified for export jobs.')
+        c.argument('export', action=AddExport, nargs='+', help='A property containing information about the blobs to be exported for an export job. This property is required for export jobs, but must not be specified for import jobs.')
+        c.argument('provisioning_state', help='Specifies the provisioning state of the job.')
```

Comment:
this is an argument?

Reply:
Nice finding. It should be a bug of swagger file. It reuse a `JobDetail` object in both `PutParameter` and `JobResponse`. As a result of that we can specify this value when create a job, however the server will ignore it. I think we should remove it from `create` command parameters.